README.md

```diff
@@ -1,44 +1,69 @@
-## scratch-www
+# scratch-www
 #### Standalone web client for Scratch
 
 [![Build Status](https://travis-ci.org/LLK/scratch-www.svg)](https://travis-ci.org/LLK/scratch-www)
 [![Coverage Status](https://coveralls.io/repos/github/LLK/scratch-www/badge.svg?branch=develop)](https://coveralls.io/github/LLK/scratch-www?branch=develop)
-[![dependencies Status](https://david-dm.org/llk/scratch-www/status.svg)](https://david-dm.org/llk/scratch-www)
-[![devDependencies Status](https://david-dm.org/llk/scratch-www/dev-status.svg)](https://david-dm.org/llk/scratch-www?type=dev) [![Greenkeeper badge](https://badges.greenkeeper.io/LLK/scratch-www.svg)](https://greenkeeper.io/)
+[![Greenkeeper badge](https://badges.greenkeeper.io/LLK/scratch-www.svg)](https://greenkeeper.io/)
 
-### Where am I?
-Physically? No idea.
+## Overview
 
-Digitally? You’re at Scratch’s open source web client!
+This is Scratch’s open source web client! This is the code for much of the [Scratch website](https://scratch.mit.edu).
 
-We’re working to update the [Scratch website](https://scratch.mit.edu) to use a new codebase, contained in this repository.
+In particular, this codebase includes code for:
+* the "project page", which shows a playable version of the project, along with the project's title, description, comments, remixes and studios; this page operates in the background when you "See inside" a project
```

Comment:
Hm, can we really include "remixes" and "studios"? View All page isn't migrated.

Reply:
Here, I'm trying to describe what the project page includes, for instance:

![image](https://user-images.githubusercontent.com/3431616/69928811-d9c79700-148a-11ea-9784-6df3f28446f2.png)

Does that make sense?

I'm not trying to make claims about what else this shows. Maybe it would be clearer if I said "thumbnails of a few remixes and studios" or something like that?